scripts/mongodb/updateCourses.js

```diff
@@ -109,12 +110,20 @@ for (var i = 0; i < courses.length; i++) {
       }
     }
   }
-  courses[i].concepts = Object.keys(concepts);
-}
+  course.concepts = Object.keys(concepts);
+
+});
+
 
 print("Updating courses..");
 for (var i = 0; i < courses.length; i++) {
-  db.courses.update({name: courses[i].name}, courses[i], {upsert: true});
+  db.courses.update({name: courses[i].name}, {$set: courses[i]}, {upsert: true});
```

Comment:
For this particular script I might not use `$set` since it might (silently) fail to remove an attribute if we ever want to remove one.